FrontEndLib/ListBoxWidget.cpp

```diff
@@ -1090,7 +1116,32 @@ void CListBoxWidget::HandleKeyDown(
 				wPrevCursorLine = (UINT)(-1); //select the current line below
 		break;
 
+		case SDLK_BACKSPACE:
+			if (this->wstrActiveFilter.size() > 0) {
+				UpdateFilter(this->wstrActiveFilter.substr(0, this->wstrActiveFilter.size() - 1));
+				RequestPaint();
+			}
+			break;
+
+		case SDLK_ESCAPE:
+			if (this->wstrActiveFilter.size() > 0) {
+				UpdateFilter(L"");
+				RequestPaint();
+				PreventBubbling();
+			}
+			break;
+
 		default:
+			if (this->bAllowFiltering && !(KeyboardEvent.keysym.mod & KMOD_SHIFT)) {
+				const WCHAR character = KeyboardEvent.keysym.sym == SDLK_SPACE ? L' ' : TranslateUnicodeKeysym(KeyboardEvent.keysym);
```

Comment:
I'm not sure that L' ' works outside of Windows on Linux and OS X.  We should probably use We(' ') instead for portability.

Reply:
I have fixed this for all the instances of L"" and L'' that I've introduced in 5.2 so far.